homeassistant/components/aurora/binary_sensor.py

```diff
@@ -125,22 +131,16 @@ def update(self):
     def get_aurora_forecast(self):
         """Get forecast data and parse for given long/lat."""
         raw_data = requests.get(URL, headers=self.headers, timeout=5).text
```

Comment:
Unfortunately I expect the requirement will be that this will need to be wrapped into a PyPi library rather than directly calling the API from the Home Assistant integration. Direct access to external APIs is no longer acceptable in Home Assistant.